kuzzleio/kuzzle-admin-console
# 858 security console error

## What does this PR do ?
fix #858 

I just deleted all the call to the old vuex module: [collection vuex module](https://github.com/kuzzleio/kuzzle-admin-console/tree/3-dev/src/vuex/modules/collection)

### How should this be manually tested?
  - Step 1 : run kuzzle
  - Step 2 : go to AC in security tab
  - Step 3 : switch between left entries (Users, Roles Profiles), should not get `receiveCollectionDetails` console error

### Other changes
/

### Boyscout
/
